descartes_moveit/include/descartes_moveit/seed_search.h

```diff
@@ -0,0 +1,31 @@
+#ifndef SEED_SEARCH_H
+#define SEED_SEARCH_H
+
+#include <moveit/robot_state/robot_state.h>
+
+namespace descartes_moveit
+{
+namespace seed
+{
+
+
+typedef std::pair<unsigned, unsigned> JointPair;
+typedef std::vector<JointPair> JointPairVec;
+typedef std::vector<double> JointConfig;
```

Comment:
I did it for brevity in the underlying code. I will make the external interface operate with the base types and just move the typedefs under the hood.